everything_else/reproducing.m

```diff
@@ -155,6 +155,13 @@
 RHS = 1/2*(b1 + b2 + b3 + b4 - 2*ba1).*(b1 + b2 + b3 + b4 - 2*ba1 - 1);
 
 %% Pg. 21, PTR-BCR-2
+
+b = dec2bin(2^6-1:-1:0)-'0';
+b1=b(:,1);b2=b(:,2);b3=b(:,3);b4=b(:,4);ba1=b(:,5);ba2=b(:,6);
+LHS = min(reshape (b1.*b2.*b3.*b4,2,[]));
+RHS = min(reshape (1/2*(4 + b1 + b2 + b3 + b4 - ba1 - 2.*ba2).*(3 + b1 + b2 + b3 + b4 - ba1 - 2.*ba2),2,[]));
+isequal(LHS,RHS);
```

Comment:
Sorry for the super late review! So this right here actually doesn't give us the correct verification because the answer after isequal(LHS, RHS) is actually 0, when we want it to be 1. My suggestion (and what I believe to be the issue, though I'm not 100% on this fact since I'm not an expert) is the fact that we're reshaping the answer. So once we reshape the multiplication, what we're doing essentially is saying that the minimum over every two pairs of answers (ie over ba1, ba2) should give the same answer, though I think it should be over all of them